isort/setuptools_commands.py

```diff
@@ -1,7 +1,7 @@
 import glob
 import os
 import sys
-from typing import Any, Dict, Iterator, List
+from typing import Any, Iterator
```

Comment:
These changes seem unrelated? Why are we removing `user_options` ?

Reply:
A bit, but for some reason mypy started complaining about it. It may be it happened because mypy got upgraded on the way.

Reply:
Should we at least make a small mention of it in the changelog?

I'm not too up to speed with what this does, but I can assume users can be surprised this is removed without a notice.

Reply:
@DanielNoord I actually returned with user_options to be extra safe and put a type ignore with a comment so mypy does not complain.